pkg/pillar/cmd/baseosmgr/handledownload.go

```diff
@@ -288,6 +288,7 @@ func checkStorageDownloadStatus(ctx *baseOsMgrContext, objType string,
 			ss.Error = ds.LastErr
 			ret.AllErrors = appendError(ret.AllErrors, "downloader",
 				ds.LastErr)
+			ss.ErrorSource = pubsub.TypeToName(types.VerifyImageStatus{})
```

Comment:
Same here.. add support for ss.SetError() ?

Reply:
sure